watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -47,6 +47,7 @@
 class ERDtype(StrEnum):
     pressure_exchanger = "pressure_exchanger"
     pump_as_turbine = "pump_as_turbine"
+    no_ERD = "no_ERD"
```

Comment:
This option should also be listed in the exception message for `erd_type_not_found` function, line 52